### ISSUE TITLE
Select which firmware do you want download

### ISSUE THREAD

**Issue by D3fau4** [D3fau4/NightFall#21]
Actually i have 4 firmwares in the server:
![image](https://user-images.githubusercontent.com/19638408/102989677-4db16300-4516-11eb-93e4-ce2520aeb236.png)
it maybe interesting to give the user the option to choose which firmware they want.

**Comment by carcaschoi** [comment]
Just have a question, what is the biggest number of firmware can the server store.

Idea: 
first solution: Automatically detect user firmware and let user can only choose the firmware which is higher(only sysnand)(avoid brick the device)

second solution: Automatically detect user firmware and let user can only  choose firmware that match fuses of switch(only sysnand)(user can downgrade safely)(avoid brick the device)

**Comment by D3fau4** [comment]
yeah but i have another problem, the bandwidth have a limit of 100GB and this mothn in 1 weak:
![image](https://user-images.githubusercontent.com/19638408/103002363-097e8c80-452f-11eb-959e-12d14986da49.png)

**Comment by Kronos2308** [comment]
@D3fau4  you can just pack the update in a zip and upload that to gdrive for example and usa a generic api key to  download the file once you made that public that way you safe bandwith
in the server just let the info file with the link to the zip
